core/src/scene/sceneLoader.cpp

```diff
@@ -971,6 +971,7 @@ void SceneLoader::loadSource(const std::shared_ptr<Platform>& platform, const st
     int32_t minDisplayZoom = -1;
     int32_t maxDisplayZoom = -1;
     int32_t maxZoom = 18;
+    int32_t tileScale = 0; // Equivalent of tileSize of 256px
```

Comment:
More comment! 'A tileScale of 0 corresponds to a tile_size of 256, 1 corresponds to 512, 2 corresponds to 1024, etc.'